src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/GetTimelineTest.kt

```diff
@@ -148,25 +176,53 @@ class GetTimelineTest : IntegrationTestBase() {
 
     // Then
     val actual = response.responseBody.blockFirst()
+    assertThat(actual.events).hasSize(6)
+    val actionPlanCreatedCorrelationId = actual.events[0].correlationId!!
+    val goalUpdatedCorrelationId = actual.events[4].correlationId!!
     assertThat(actual)
       .isForPrisonNumber(prisonNumber)
       .event(1) {
         it.hasEventType(TimelineEventType.ACTION_PLAN_CREATED)
           .hasPrisonId("BXI")
           .hasSourceReference(actionPlanReference.toString())
           .hasNoContextualInfo() // creating an action plan has no contextual info
+          .hasCorrelationId(actionPlanCreatedCorrelationId)
       }
       .event(2) {
+        it.hasEventType(TimelineEventType.GOAL_CREATED)
+          .hasPrisonId("BXI")
+          .hasSourceReference(goal1Reference.toString())
+          .hasContextualInfo("Learn German")
+          .hasCorrelationId(actionPlanCreatedCorrelationId)
+      }
+      .event(3) {
         it.hasEventType(TimelineEventType.GOAL_CREATED)
           .hasPrisonId("BXI")
           .hasSourceReference(goal2Reference.toString())
           .hasContextualInfo("Learn French")
+          .doesNotHaveCorrelationId(actionPlanCreatedCorrelationId)
```

Comment:
I cant think of a better name right now, but `doesNotHaveCorrelationId` made me think initially that you were expecting it to not have a correlation ID at all, rather than "it has a correlationID, but just not _this_ one" 🤔

Reply:
`.correlationIdIsNotEqualTo()`?